core/src/main/scala/higherkindness/mu/rpc/srcgen/Generator.scala

```diff
@@ -38,5 +42,5 @@ trait Generator {
   protected def generateFrom(
       inputFile: File,
       serializationType: Model.SerializationType
-  ): Option[(String, Seq[String])]
+  ): Option[(String, Seq[ErrorOr[String]])]
```

Comment:
Shouldn't this be `ErrorOr[Seq[String]]`? The `Seq[String]` is the file content, right? i.e. a list of lines of text. So we want to return either a list of errors we found in the input file, or a valid list of lines to write to the output file.

Reply:
Hmmm maybe I'm thinking about this backward but I think it should be a `Seq[ErrorOr[String]]` because there can be multiple errors in a given file and I want to return a list of those if needed.  So to me, the type param `Seq[ErrorOr[String]]` means that this method returns a `Seq` of errors and/or a `Seq` of strings.  It's possible that I'm interpreting this type parameter incorrectly.

Reply:
Talked with Chris offline about this and he's right; it should be an `ErrorOr[Seq[String]]` (I confused myself with my type alias, shoulda called it `ErrorsOr`.